.github/workflows/cron-deploy-to-production.yml

```diff
@@ -0,0 +1,23 @@
+name: "Automatic production deployment"
+
+on:
+  schedule:
+    # “At 09:00 on every day-of-week from Tuesday through Friday.” 
+    -   cron: "0 9 * * 2-5"
+  workflow_dispatch:
+
+jobs:
+  backup:
+    runs-on: ubuntu-latest
+    name: Force-push to the production branch
+    steps:
+    - name: sync
+      env:
+        # Uses `github.token`: the default, ephemeral token automatically generated by GitHub
+        # https://docs.github.com/en/actions/security-guides/automatic-token-authentication#modifying-the-permissions-for-the-github_token
+        PERSONAL_ACCESS_TOKEN: ${{ github.token }}
+      run: |
+        set -xe
+        git clone "https://theopensystemslab:${PERSONAL_ACCESS_TOKEN}@github.com/planx-new.git" planx-new
+        cd planx-new || exit 1
+        git push --force origin origin/main:production
```

Comment:
with our recently updated `production` branch protection rules, i'd expect this `push` command to fail because it will be lacking an "approval" ? i think that perhaps a few of our next steps here have turned to be a bit contradictory to each other :upside_down_face:

Reply:
Yeah let's discuss this


![image](https://user-images.githubusercontent.com/7684574/208908835-42b3ef8e-9d2e-42ea-95e5-0e9a1270b63a.png)